js/social-comments.js

```diff
@@ -1,23 +1,91 @@
-const socialComments = document.querySelector('.social__comments');
+const blockBigPicture = document.querySelector('.big-picture'); //модальное окно с фото и информацией о нем
+const socialCommentsCount = blockBigPicture.querySelector('.social__comment-count'); //сообщение о количестве комментариев
+const commentsCount = socialCommentsCount.querySelector('.comments-count'); //число комментариев к фото
+const printedCommets = socialCommentsCount.querySelector('.printed-comments'); //отображение в разметке числа напечатанных комментариев
 
-const commentTemplate = document.querySelector('#comment').content; //шаблон комментария
+const socialComments = document.querySelector('.social__comments'); //список комментариев к фото
+const inputComment = blockBigPicture.querySelector('.social__footer-text'); //поле ввода комментария
+const buttonDownloadComments = blockBigPicture.querySelector('.social__comments-loader'); //кнопка загрузки новых комментариев
 
+const commentTemplate = document.querySelector('#comment').content; //шаблон комментария
 const commentTemplateListElement = commentTemplate.querySelector('.social__comment'); //элемент списка комментариев, потомок шаблона
+const fragment = document.createDocumentFragment();
+
+const COUNT_COMMENTS_TO_DOWNLOAD = 5;
+
+let photoData = [];
+let countComments = 0;
+
+function buttonDownloadCommentsClickHandler () {
+  downloadComments();
+}
 
 function cleanListComments () {
   while (socialComments.firstChild) {
     socialComments.removeChild(socialComments.firstChild);
   }
 }
 
-function createFragmentComments (data) {
-  cleanListComments(); //очистка списка комментариев
+function downloadComments() {
+  const countPrintedComments = socialComments.children.length; //сколько уже загружено комментов
+  const residueComments = countComments - countPrintedComments;
```

Comment:
```suggestion
  const restOfComments = countComments - countPrintedComments;
```

слово резидью никто не использует